src/Akeneo/Pim/Structure/Bundle/Doctrine/ORM/Repository/ExternalApi/FamilyRepository.php

```diff
@@ -101,14 +95,24 @@ protected function addFilters(QueryBuilder $qb, array $searchFilters): QueryBuil
                 switch ($criterion['operator']) {
                     case 'IN':
                         $qb->andWhere($qb->expr()->in($field, $parameter));
+                        $qb->setParameter($parameter, $criterion['value']);
                         break;
                     case '>':
                         $qb->andWhere($qb->expr()->gt($field, $parameter));
+                        $qb->setParameter($parameter, $criterion['value']);
+                        break;
+                    case '=':
+                        if ('has_products' === $property) {
+                            $qb->andWhere($qb->expr()->in('r.id', 'family_ids_used_by_products'));
+                            $qb->setParameter(
+                                'family_ids_used_by_products',
+                                implode(',', $this->getFamilyIdsUsedByProductsQuery->execute())
```

Comment:
```suggestion
                                implode(',', $this->getFamilyIdsUsedByProductsQuery->execute())
```

Should be replace by exists.